src/components/EventBanner/EventBanner.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "./EventBanner.scss";
+import dayjs from "dayjs";
+import * as SEC from "../SEC/lib.js";
+
+interface EventBannerProps {
+	flyerSource: any;
+	altText: string;
+	title: string;
+	desc: string;
+	startDate: dayjs.Dayjs;
+	endDate: dayjs.Dayjs;
+	meetingLink: string;
+	isGbm: boolean;
+	now?: dayjs.Dayjs;
+}
+
+const EventBanner = (props: EventBannerProps): JSX.Element | null => {
+	let now: dayjs.Dayjs = props.now ? props.now : dayjs();
+
+	let flyer = props.flyerSource ? require(`../../assets/flyers/${props.flyerSource}`) : null;
+	let imageElement = flyer ? (
+		<img
+			loading="lazy"
+			src={flyer.default}
+			alt={
+				props.altText
+					? `${props.altText}`
+					: `Flyer for ${props.title ? `${props.title} ` : ""}event`
```

Comment:
`altText` and `title` are marked as required props. We should either mark these as optional or remove these existence checks.